### PR TITLE
Prompt before deleting lists

### LINKED ISSUE #1998: Ask the user before removing a list
It’s easy to remove a list in Tusky by taping the wrong menu option, and it’s hard to remake from memory. Tusky should ask the user if they’re sure they want to remove a list, like in Mastodon Web UI.

* * * *
- Tusky Version: 12.1 and 13.0 beta 1-8ec804f8
- Android Version: 9
- Android Device: Oneplus 5 (cheeseburger)

### PR DESCRIPTION
**Pull request by Tak**
Fixes #1998

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -592,5 +592,6 @@
     <string name="wellbeing_hide_stats_posts">Hide quantitative stats on posts</string>
     <string name="wellbeing_hide_stats_profile">Hide quantitative stats on profiles</string>
     <string name="error_upload_max_media_reached">You cannot upload more than %1$d media attachments.</string>
+    <string name="dialog_delete_list_warning">Delete %s?</string>
```
Comment (connyduck) [inline app/src/main/res/values/strings.xml:595]: Do you really want to delete the list %s?

File: app/src/main/java/com/keylesspalace/tusky/ListsActivity.kt [hunk 2]
```diff
@@ -143,6 +143,18 @@ class ListsActivity : BaseActivity(), Injectable, HasAndroidInjector {
         editText.text?.let { editText.setSelection(it.length) }
     }
 
+    private fun showListDeleteDialog(list: MastoList) {
+        AlertDialog.Builder(this)
+                .setMessage(getString(R.string.dialog_delete_list_warning, list.title))
+                .setPositiveButton(R.string.action_delete){ _, _ ->
+                    viewModel.deleteList(list.id)
+                }
+                .setNegativeButton(android.R.string.cancel) { d, _ ->
```
Comment (connyduck) [inline app/src/main/java/com/keylesspalace/tusky/ListsActivity.kt:152]: `.setNegativeButton(android.R.string.cancel, null)`

(change it for the other dialog as well, please)

**Comment by Tak** [comment]
:heavy_check_mark:

**Review by connyduck — APPROVED** [review]

**Comment by ariasuni** [comment]
Thanks! Hopefully I won’t delete any list accidentally with this change :)

**Pull request merged**